### PR TITLE
Fixing the structure of RESTProxy documentation:  Updating the OperationId

### PR DESCRIPTION
**Pull request by anushreeringne**
Updating the OperationId because the grouping of operations isn't correct.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "tags": [ "v1consumer" ],
         "summary": "Consume records using a simple consumer",
         "description": "Consume records from one partition of a topic beginning with a specific offset. By default count is 1. It can read maximum of 6 megabytes of data at once",
-        "operationId": "consumeTopicInPartitionWithOffset",
+        "operationId": "Consumer_ConsumeTopicInPartitionWithOffset",
```
Comment (anushreeringne) [inline specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json]: Please help me fix the Prettier failure. I am not sure what is incorrect and how to fix it.

**Comment by hrasheed-msft** [comment]
@anushreeringne There are a few tips at this link: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md on checking the json syntax:

Please run the following command:
npm install; npm run prettier-fix
Or if you want to fix specified service:
npm install; npm run prettier -- --write "specification/<service>/**/*.json"
Then please commit and push changes made by prettier.
Let me know if that doesn’t turn up anything.

**Comment by hrasheed-msft** [comment]
@zikalino @MyronFanQiu could you offer any guidance on why the prettiercheck might be failing? We tried the module, but didnt get any errors.

**Comment by NullMDR** [comment]
@anushreeringne Is this spec management plane or data plane?

For the prettier fix please follow the exact command:

npm install; npm run prettier -- --write "specification/hdinsight/**/*.json"

**Comment by anushreeringne** [comment]
> @anushreeringne Is this spec management plane or data plane?
> 
> For the prettier fix please follow the exact command:
> 
> moba

This is data plane. As I mentioned I did try to run prettier but was unable to fix the errors. 

> @anushreeringne Is this spec management plane or data plane?
> 
> For the prettier fix please follow the exact command:
> 
> npm install; npm run prettier -- --write "specification/hdinsight/**/*.json"

Hi, I got the following error when I tried to run it. I am running it on "Ubuntu 16.04.6 LTS"" and I did update the latest node.js and npm.

npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@2.1.2
npm ERR! Linux 4.15.0-1066-azure
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! autorest@2.0.4413 preinstall: `node ./preinstall-check`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the autorest@2.0.4413 preinstall script 'node ./preinstall-check'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the autorest package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./preinstall-check
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs autorest
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls autorest
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/sshuser/azure-rest-api-specs/npm-debug.log

> azure-rest-api-specs-tests@0.1.0 prettier /home/sshuser/azure-rest-api-specs
> prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"

sh: prettier: command not found

npm ERR! Linux 4.15.0-1066-azure
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "prettier" "--" "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! azure-rest-api-specs-tests@0.1.0 prettier: `prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the azure-rest-api-specs-tests@0.1.0 prettier script 'prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the azure-rest-api-specs-tests package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs azure-rest-api-specs-tests
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls azure-rest-api-specs-tests
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/sshuser/azure-rest-api-specs/npm-debug.log

**Comment by NullMDR** [comment]
Oh please rebase on latest master


Best Regards,
Phoenix He

________________________________

**Comment by anushreeringne** [comment]
> Oh please rebase on latest master Best Regards, Phoenix He
> […](#)
> ________________________________ From: anushreeringne <notifications@github.com> Sent: Friday, February 21, 2020 5:21:05 PM To: Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com> Cc: Phoenix He <Phoenix.He@microsoft.com>; Comment <comment@noreply.github.com> Subject: Re: [Azure/azure-rest-api-specs] Fixing the structure of RESTProxy documentation: Updating the OperationId (#8327) @anushreeringne<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fanushreeringne&data=02%7C01%7CPhoenix.He%40microsoft.com%7C8c104fb1405846fc3c8408d7b6af60fc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637178736668968033&sdata=hixkV5gE8vDp21qBqzDvNfmdZn2KhD%2BOWHfUNV7H6Kc%3D&reserved=0> Is this spec management plane or data plane? For the prettier fix please follow the exact command: moba This is data plane. As I mentioned I did try to run prettier but was unable to fix the errors. @anushreeringne<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fanushreeringne&data=02%7C01%7CPhoenix.He%40microsoft.com%7C8c104fb1405846fc3c8408d7b6af60fc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637178736668973023&sdata=lcuqXmc9xvX%2F1kDUv86nS2Eu%2BG6O%2F64fhMCxtM%2Bw2ZM%3D&reserved=0> Is this spec management plane or data plane? For the prettier fix please follow the exact command: npm install; npm run prettier -- --write "specification/hdinsight/**/*.json" Hi, I got the following error when I tried to run it. I am running it on "Ubuntu 16.04.6 LTS"" and I did update the latest node.js and npm. npm WARN optional Skipping failed optional dependency /chokidar/fsevents: npm WARN notsup Not compatible with your operating system or architecture: fsevents@2.1.2 npm ERR! Linux 4.15.0-1066-azure npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" npm ERR! node v4.2.6 npm ERR! npm v3.5.2 npm ERR! file sh npm ERR! code ELIFECYCLE npm ERR! errno ENOENT npm ERR! syscall spawn npm ERR! autorest@2.0.4413 preinstall: node ./preinstall-check npm ERR! spawn ENOENT npm ERR! npm ERR! Failed at the autorest@2.0.4413 preinstall script 'node ./preinstall-check'. npm ERR! Make sure you have the latest version of node.js and npm installed. npm ERR! If you do, this is most likely a problem with the autorest package, npm ERR! not with npm itself. npm ERR! Tell the author that this fails on your system: npm ERR! node ./preinstall-check npm ERR! You can get information on how to open an issue for this project with: npm ERR! npm bugs autorest npm ERR! Or if that isn't available, you can get their info via: npm ERR! npm owner ls autorest npm ERR! There is likely additional logging output above. npm ERR! Please include the following file with any support request: npm ERR! /home/sshuser/azure-rest-api-specs/npm-debug.log azure-rest-api-specs-tests@0.1.0 prettier /home/sshuser/azure-rest-api-specs prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json" sh: prettier: command not found npm ERR! Linux 4.15.0-1066-azure npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "prettier" "--" "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json" npm ERR! node v4.2.6 npm ERR! npm v3.5.2 npm ERR! file sh npm ERR! code ELIFECYCLE npm ERR! errno ENOENT npm ERR! syscall spawn npm ERR! azure-rest-api-specs-tests@0.1.0 prettier: prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json" npm ERR! spawn ENOENT npm ERR! npm ERR! Failed at the azure-rest-api-specs-tests@0.1.0 prettier script 'prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json"'. npm ERR! Make sure you have the latest version of node.js and npm installed. npm ERR! If you do, this is most likely a problem with the azure-rest-api-specs-tests package, npm ERR! not with npm itself. npm ERR! Tell the author that this fails on your system: npm ERR! prettier "--write" "specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json" npm ERR! You can get information on how to open an issue for this project with: npm ERR! npm bugs azure-rest-api-specs-tests npm ERR! Or if that isn't available, you can get their info via: npm ERR! npm owner ls azure-rest-api-specs-tests npm ERR! There is likely additional logging output above. npm ERR! Please include the following file with any support request: npm ERR! /home/sshuser/azure-rest-api-specs/npm-debug.log — You are receiving this because you commented. Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F8327%3Femail_source%3Dnotifications%26email_token%3DAAR4YQYQMBXE7PNB5X5BBLTRD6MIDA5CNFSM4KRDOIIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMSB2XA%23issuecomment-589569372&data=02%7C01%7CPhoenix.He%40microsoft.com%7C8c104fb1405846fc3c8408d7b6af60fc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637178736668978008&sdata=urynq7nSaEGSVpoGUTcEEDdW%2FnYX%2Bs7pIhcNqcvjBmI%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAR4YQ4XQTWLQBC5D7T4ZDTRD6MIDANCNFSM4KRDOIIA&data=02%7C01%7CPhoenix.He%40microsoft.com%7C8c104fb1405846fc3c8408d7b6af60fc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637178736668983004&sdata=30AaN0HEoFygqE3dV5JUM3wal54x7JiaMzavJJzU3RM%3D&reserved=0>.

Rebased

**Comment by anushreeringne** [comment]
Reopening to triggerbuild

**Comment by anushreeringne** [comment]
#sign-off

**Review by PhoenixHe-NV — APPROVED** [review]

**Pull request merged**